### DISCUSSION TITLE
CarDemo cars rolling over, lower center of mass possible?

### DISCUSSION THREAD

**Discussion by Armynator** [bepu/bepuphysics2#315] [Q&A]
If the speed is too high or the forwardForce is increased for the cars, they tend to roll over/flip very quickly in the slightest curves. What would be the easiest way to avoid that? in other engine lowering the center of mass seems to help with this problem, but I couldn't see any quick way to override the center of mass in the car demo?

**Comment by RossNordby** [comment]
There is no universal center of mass offset; instead, `Compound` and `BigCompound` shapes allow you to specify per-child offsets. Offsetting the center of mass is equivalent to offsetting the collision shape from the center of mass.

`Compound` has no acceleration structure, so if you have very few child shapes (or just one), it's an extremely low overhead option.
